esmvaltool/main.py

```diff
@@ -107,6 +104,7 @@ def read_config_file(config_file, namelist_name):
         'output_file_type': 'ps',
         'output_dir': './output_dir',
         'save_intermediary_cubes': False,
+        'parallel': True,
```

Comment:
Is that the default setting? I would propose setting it to `False`

Reply:
@nielsdrost had an even better idea of making the number of tasks that can be run in parallel configurable. Of course we can set that to 1 by default. Shall I change that?

Reply:
Good idea!